.github/workflows/audit.yaml

```diff
@@ -1,11 +1,25 @@
-name: Audit Check
-on: [pull_request]
+name: Security Audit
+on:
+  pull_request:
+    paths: Cargo.lock
+  push:
+    branches: develop
+    paths: Cargo.lock
+  schedule:
+    - cron: '0 0 * * *'
 
 jobs:
   security_audit:
+    name: Security Audit
     runs-on: ubuntu-latest
     steps:
-      - uses: actions/checkout@v2
+      - uses: actions/checkout@v1
```

Comment:
Is there a particular reason this uses `v1`?

Reply:
No, that's a mistake on my part, I blindly copy-pasted the config file used above. Will change it back to v2. Thanks for noticing!